pages/profile/index.js

```diff
@@ -7,7 +7,9 @@ import styled from "styled-components";
 import NavBar from "../../components/NavBar/index";
 import LibraryLink from "../../components/LibraryLink";
 
-export default function ProfilePage() {
+export default function ProfilePage({ libraryGames, wishlistGames }) {
+  const libraryCount = libraryGames.length;
```

Comment:
you only need the length of this array here.
It would be possible to only give `amountOfLibraryGames` and `amountOfWhishListGames` to this component.